### PR TITLE
Add events job to main_summary dag

### PR DESCRIPTION
**Pull request by sunahsuh**
Landed this job last week in telemetry-batch-view and am now getting around to adding it to the daily main_summary dag (I'll run the backfill by hand through whenever this lands)

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-airflow/87)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: dags/main_summary.py [hunk 1]
```diff
@@ -62,7 +62,17 @@
                       output_visibility="private",
                       dag=dag)
 
+t6 = EMRSparkOperator(task_id="main_events",
```
Comment (vitillo) [inline dags/main_summary.py]: Please add yourself as owner.

File: dags/main_summary.py [hunk 2]
```diff
@@ -62,7 +62,17 @@
...
+                      job_name="Main Events View",
+                      execution_timeout=timedelta(hours=4),
+                      release_label="emr-5.2.1",
+                      instance_count=1,
```
Comment (vitillo) [inline dags/main_summary.py]: Is 1 machine enough?

Comment (sunahsuh) [inline dags/main_summary.py]: Yes -- currently this job is taking about 10 minutes per day. I'm sure that'll go up as more people adopt events but for now this is fine.

**Comment by sunahsuh** [comment]
Updated with the requested changes

**Comment by vitillo** [comment]
Please redeploy.

**Pull request merged**